manim/scene/scene.py

```diff
@@ -827,7 +830,12 @@ def wrapper(self, *args, **kwargs):
                     file_writer_config["skip_animations"] = True
             else:
                 hash_play = "uncached_{:05}".format(self.num_plays)
-                self.play_hashes_list.append(hash_play)
+            self.animations_hashes.append(hash_play)
+            self.file_writer.add_partial_movie_file(hash_play)
+            logger.debug(
+                "Animations hashes list of the scene : (concatened to 5) %(h)s",
+                {"h": str(self.animations_hashes[:5])},
```

Comment:
Is there a reason for the magic number 5? Is there any harm in printing all of them?

Also, there is a typo in `concatenated` (should it really be concatenated, and not something like shortened...?), maybe as an alternative suggestion:

```suggestion
                "List of the first few animation hashes of the scene: %(h)s",
                {"h": str(self.animations_hashes[:5])},
```

Reply:
Thanks for that ! 

And 5 is an arbitraty number. I did this in order to not overflood the logs.